runtime/taskcontext.go

```diff
@@ -55,6 +60,7 @@ type TaskContext struct {
 	webHookSet *webhookserver.WebHookSet
 	logStream  *stream.Stream
 	mu         sync.RWMutex
+	queue      *queue.Queue
```

Comment:
I already have an interface used elsewhere so that I can mock it out when needed, I plan to just move it here into the runtime